events/python-programming-workshop/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="shortcut icon" href="../../images/favicon.ico" type="image/x-icon">
+    <title>Web Designing</title>
+    <meta name="description" content="GLUG PACE is organizing Web Designing workshop for the students of second and third year B.E in P.A. College of engineering.">
+    <link rel="stylesheet" href="../../css/style.css">
+</head>
+<body>
+    <!-- Navbar -->
+    <header>
+        <div class="container">
+            <a href='/' id="logo">GLUG PACE</a>
+            <ul class="navbar-links">
+                <li class="nav-link"><a href='/events/'>Events</a></li>
+                <li class="nav-link"><a href='/blog/'>Blog</a></li>
+                <li class="nav-link"><a href='/team/'>Team</a></li>
+            </ul>
+        </div>
+    </header>
+
+    <!-- Read post section -->
+    <div class="container padding">
+        <div class="data-content">
+            <img class="data-image" src="images/python-programming-workshop.jpeg" alt="" />
+            <div class="data-desc">
+                <h1 class="data-heading">Python Programming Workshop</h1>
+                <p class="data-date">Date: 15-03-2019</p>
+                <p class="data-venue">Venue: UNIX LAB</p> 
+                <hr>
+                <div class="data-body">
+                    <p>The workshop will cover basics of python programming language.</p>
```

Comment:
Can you add a bit more description like!
The workshop will help you in getting started with Python Programming. This workshop is for the beginners with basic programming knowledge.